### PR TITLE
New endpoint added and documentation

### PR DESCRIPTION
**Pull request by Melic93**
New endpoint added: GET /{name}
It returns a personalised welcome message saying "Hello {name}"

Some code documentation added on HelloController.java

### CODE REVIEW

File: src/main/java/es/unizar/webeng/hello/HelloController.java [hunk 1]
```diff
@@ -12,10 +13,29 @@
...
+     * This annotation is used to map the welcome function to a GET Request on path: "/"
+     * @return a String composed with current date + a "Hello World" message
+     */
+
     @GetMapping("/")
     public String welcome(Map<String, Object> model) {
         model.put("time", new Date());
         model.put("message", message);
         return "wellcome";
     }
```
Comment (fjlopez) [inline src/main/java/es/unizar/webeng/hello/HelloController.java]: Fix typo "welcome"

File: src/main/java/es/unizar/webeng/hello/HelloController.java [hunk 2]
```diff
@@ -12,10 +13,29 @@
     @Value("${app.message:Hello World}")
     private String message;
 
+    /**
+     * 
+     * This annotation is used to map the welcome function to a GET Request on path: "/"
+     * @return a String composed with current date + a "Hello World" message
+     */
+
```
Comment (fjlopez) [inline src/main/java/es/unizar/webeng/hello/HelloController.java]: Remow whitespace

File: src/main/java/es/unizar/webeng/hello/HelloController.java [hunk 3]
```diff
@@ -12,10 +13,29 @@
...
+     * 
+     * This annotation is used to map the personalWelcome function to a GET Request on path: "/{name}"
+     * @PathVariable annotation is used to extract a variable from the url
+     * @return a String saying "Hello {name}"
+     */
+    @GetMapping("/{name}")
+    public String personalWelcome(Map<String, Object> model, @PathVariable String name) {
+        model.put("name", "Hola " + name);
+        return "wellcome";
+    }
```
Comment (fjlopez) [inline src/main/java/es/unizar/webeng/hello/HelloController.java]: fix typo "welcome" (I know, it was the name of an existing file ... rename it too!)

File: README.md [hunk 4]
```diff
@@ -5,4 +5,5 @@ Please, go to the [Wiki](https://github.com/UNIZAR-30246-WebEngineering/lab1-git
 User name | NIA | Travis-CI|Score
 ----------|-----|----------|-----
 [UNIZAR-30246-WebEngineering](https://github.com/UNIZAR-30246-WebEngineering/lab1-git-race) |30246 | [![Build Status](https://travis-ci.org/UNIZAR-30246-WebEngineering/lab1-git-race.svg)](https://travis-ci.org/UNIZAR-30246-WebEngineering/lab1-git-race)
-your name | your nia | your Travis-ci status
+Víctor Fernández Melic | 651943 | [![Build Status](https://travis-ci.org/Melic93/lab1-git-race.svg)](https://travis-ci.org/Melic93/lab1-git-race)
```
Comment (fjlopez) [inline README.md]: Sync with the latest changes.

File: src/main/java/es/unizar/webeng/hello/HelloController.java (hunk 2 above)
Comment (Melic93) [reply src/main/java/es/unizar/webeng/hello/HelloController.java]: I don't know what you mean with whitespace in that piece of code

Comment (fjlopez) [reply src/main/java/es/unizar/webeng/hello/HelloController.java]: Remove the space between the Javadoc comment and the method declaration.

**Comment by fjlopez** [comment]
Fix the conflicts with `README.md`, please.

**Pull request merged**